src/app/menu/menu.component.scss

```diff
@@ -103,7 +103,7 @@
 }
 
 .prevent-inner-circle {
-  background: $prevent-color;
+    background: $malaria-color;
```

Comment:
no this is not what I meant, generic colors in the sense define the color and use it here.. like
in scss file $red : #FF0000, use $red here.

Reply:
@lunayach you can add variable names like light red, maroon, etc for custom colors of the menu page. That can be done right? 
@shreyans29 shall we take that task as a separate refactoring issue?